data/dev.tchx84.Portfolio.desktop.in

```diff
@@ -7,4 +7,4 @@ Categories=Utility;
 Icon=dev.tchx84.Portfolio
 StartupNotify=true
 X-Purism-FormFactor=Mobile;
-Keywords=folder;manager;explore;disk;filesystem;
+Keywords=folder;file;manager;explore;browse;disk;filesystem;
```

Comment:
`file` is redundant with the description, e.g. try searching for "file" in GNOME shell (or whatever DE you're using) and you will notice that Portfolio already shows up.

"browser" is used specifically for web browsers which could be confusing. E.g, try searching for "browse" and you will see that no other file manager shows up.

A few resources:

* https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html (see comments about redundancy).
* https://gitlab.gnome.org/GNOME/nautilus/-/blob/master/data/org.gnome.Nautilus.desktop.in.in (nautilus)
* https://invent.kde.org/system/dolphin/-/blob/master/src/org.kde.dolphin.desktop (dolphin)

But since we're looking at this, how about adding the entries that are missing? e.g. `GenericName` and other `Categories`. See the examples above.